src/mesh/NodeDB.cpp

```diff
@@ -1479,6 +1479,8 @@ void NodeDB::updatePosition(uint32_t nodeId, const meshtastic_Position &p, RxSou
     info->has_position = true;
     updateGUIforNode = info;
     notifyObservers(true); // Force an update whether or not our node counts have changed
+
```

Comment:
o
src/mesh/NodeDB.cpp: In member function 'void NodeDB::updatePosition(uint32_t, const meshtastic_Position&, RxSource)':
src/mesh/NodeDB.cpp:1483:5: error: 'nimbleBluetooth' was not declared in this scope